### PR TITLE
[CPU] [ARM] HardSigmoid decompoition & JIT exec

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [ARM] HardSigmoid decompoition & JIT exec*
 - dependency: https://github.com/openvinotoolkit/openvino/pull/24165

### Tickets:
 - *CVS-139363*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@xuchen-intel Please take a look

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp [hunk 1]
```diff
@@ -160,6 +160,48 @@ class jit_exp_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const;
 };
 
+class jit_maximum_emitter : public jit_emitter {
```
Comment (xuchen-intel) [inline src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp:163]: Minor: This PR seems to denpend on a commit in your another PR. Perhaps let's add:
```
Prerequisites:
https://github.com/openvinotoolkit/openvino/pull/24165
```
in the description section of this PR.

**Review by xuchen-intel — CHANGES_REQUESTED** [review]

File: src/common/transformations/src/transformations/op_conversions/hard_sigmoid_decomposition.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/relu.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+ov::pass::HardSigmoidDecomposition::HardSigmoidDecomposition() {
+    MATCHER_SCOPE(HardSigmoidDecomposition);
+    // Decomposes HSigmoid(x) op into sub-graph (min(Relu(x + 3), 6) * const(1/6)
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/common/transformations/src/transformations/op_conversions/hard_sigmoid_decomposition.cpp]: done, thanks

File: src/common/transformations/src/transformations/op_conversions/hard_sigmoid_decomposition.cpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/hard_sigmoid.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/relu.hpp"
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/common/transformations/src/transformations/op_conversions/hard_sigmoid_decomposition.cpp]: done, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.hpp:163]: done, thanks

**Review by xuchen-intel — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**